bin/datatool/src/util.rs

```diff
@@ -77,12 +77,15 @@ fn exec_genxpriv(cmd: SubcXpriv, ctx: &mut CmdContext) -> anyhow::Result<()> {
     let xpriv = gen_priv(&mut ctx.rng, ctx.bitcoin_network);
     let mut buf = xpriv.encode();
     let mut s = base58::encode_check(&buf);
-    fs::write(&cmd.path, s.as_bytes())?;
+    let result = fs::write(&cmd.path, s.as_bytes());
```

Comment:
This approach should be fine. I think a `Zeroizing` wrapper is cleaner, but that's just a preference and a bit of a nit.